### PR TITLE
Issue #2271. Use a regular expression to get image upload data URI.

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout 

(not really sure I can write a test for this... best solution would be to get existing tests to run in safari #1704)

### CODE REVIEW

**Comment by miketaylr** [comment]
Here's an issue I just reported on my iPad: https://staging.webcompat.com/issues/1475

File: webcompat/static/js/lib/bugform.js [hunk 1]
```diff
@@ -528,6 +528,28 @@ function BugForm() {
     );
   }, this);
 
+  /*
+    Grab the data URI portion inside of a serialized data URI
+    backgroundImage, i.e, for the following two possible strings,
+    'url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt)'
+    'url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt")'
+    we expect "iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt" to be captured
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: Oops, this comment is inaccurate. Will push a fix.

**Comment by magsout** [comment]
Safari mac os: ✔️ 
Safari iPhone: ✔️

**Comment by magsout** [comment]
r+ @miketaylr

**Comment by miketaylr** [comment]
thanks!

**Pull request merged**